flame/main.cpp

```diff
@@ -45,9 +45,9 @@ static const char USAGE[] =
       -r RECORD        The base record to use as the DNS query for generators [default: test.com]
       -T QTYPE         The query type to use for generators [default: A]
       -f FILE          Read records from FILE, one per row, QNAME TYPE
-      -p PORT          Which port to flame [default: 53]
+      -p PORT          Which port to flame [defaults: 53, 853 for tcptls]
```

Comment:
The breakage is deliberate. As far as I could find, there is no way to specify two difference default values depending on another setting. So I had to avoid using 'default:' but I wanted to keep the same overall style of usage.